chroma-manager/tests/integration/utils/chroma_log_collector.py

```diff
@@ -120,67 +142,97 @@ def fetch_log(self, server, source_log_path, destination_log_filename):
         return None
 
     def fetch_log_dir(self, server, dir):
-        """
-        Collect the log directory from the target
+        """Collect the log directory from the target.
+
         :return: None on success or error message that can be used for diagnosing what went wrong.
+
         """
         logs = shell_run(['ssh', server, "ls %s | xargs -n1 basename" % dir])
 
         if logs.rc:
             return "Failed fecthing log dir %s from %s" % (server, dir)
 
         for log in logs.stdout.split():
-            destination_log_filename = "%s-%s-%s" % (server, dir.strip('/').split('/')[-1], log)
-            self.fetch_log(server, "%s/%s" % (dir, log), destination_log_filename)
+            destination_log_filename = "%s-%s-%s" % (
+                server, dir.strip('/').split('/')[-1], log)
```

Comment:
I think `os.path.basename(dir)` is the portable (and cleaner looking) version of `dir.strip('/').split('/')[-1]`.